client/app/components/atoms/Button/__tests__/Button.test.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+// import { createShallow } from '@material-ui/core/test-utils';
+
+import Button from '../Button';
+
+describe('<Button/>', () => {
+  it('renders correctly', () => {
```

Comment:
```suggestion
  it('renders correctly with props', () => {
```
There are two "it" methods (at line no.8 and at line no.18) with same definition, which i think should be different.